app/react/Library/helpers/requestState.js

```diff
@@ -50,17 +50,18 @@ export function processQuery(params, globalResources, key) {
   };
 }
 
-export default function requestState(request, globalResources, calculateTableColumns = false) {
+export default function requestState(
+  request,
+  globalResources,
+  options = { calculateTableColumns: false, geolocation: false }
+) {
   const docsQuery = processQuery(request.data, globalResources, 'library');
+
   const documentsRequest = request.set(
     tocGenerationUtils.aggregations(docsQuery, globalResources.settings.collection.toJS())
   );
 
-  const templatesWithGeolocation = globalResources.templates.find(template =>
-    template.get('properties').find(property => property.get('type') === 'geolocation')
-  );
-
-  const markersRequest = templatesWithGeolocation
+  const markersRequest = options.geolocation
```

Comment:
There is a slight chance in logic here: previously, this happened always, but it only happened if there was, at least, one template with geolocation.  Now, we are doing the request only on map, but it happens even if there isn't a single geolocation property configured.  Are we sure we don't want to also limit this to not happen if there isn't a single geolocation property?

Reply:
Users are not be able to navigate to the map view if there are no templates with geolocation. They could manually input the map view URL to go there (in that case the query would happen). I wouldn't think is necessary to also check in `requestState` if there are templates with geolocation, since the query for geolocations only happens in that map view, I thought it was like checking twice.